cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -363,6 +363,14 @@ spec:
         - name: ROOK_CEPH_STATUS_CHECK_INTERVAL
           value: "60s"
 
+        # The interval to check the health of the ceph rgw endpoints and update the status in the custom resource.
+        - name: ROOK_CEPH_OBJECT_STATUS_CHECK_INTERVAL
```

Comment:
I see these settings being on the object CR instead of in the operator.yaml. This has the benefits that the settings can be different for each object store. The settings are really specific to a single object store and there isn't a reason the operator needs a general setting for this. If the purpose is to follow the pattern of other settings in the operator.yaml, some of those settings were actually done incorrectly. For example, settings around mon failover should have been in the cluster.yaml. Only settings that aren't related to a CR should be added to the operator settings.

Reply:
Yes, I guess, I followed this "incorrect" patter. Ok, so I'll do the change in the objectstore CR and we can do a follow-up PR for the others.

Reply:
Let's discuss this more before changing the settings already in operator.yaml. Things like the mon health interval check might be overkill to put in the cluster CR. But I do think the bucket health check setting still makes sense in that CR.